classes/person.rb

```diff
@@ -8,13 +8,18 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals < rental
+  end
```

Comment:
- [ ] Great job so far, in order to achieve the` has-many` association the person itself needs to be sent. Kindly fix that.

`i.e  Rental.new(....)`